lib/modules/k2s/k2s.cluster.module/upgrade/upgrade.module.psm1

```diff
@@ -150,7 +150,6 @@ function Export-NotNamespacedResources {
                 .items[].metadata.uid,
                 .items[].metadata.selfLink,
                 .items[].metadata.creationTimestamp,
```

Comment:
What was the reason to remove annotations here?

Reply:
This topic came up during evaluation of Openebs provisioner e.g. the storage class uses annotations to specify quota enforcement:

```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  annotations:
    cas.openebs.io/config: |
      - name: StorageType
        value: "hostpath"
      - name: BasePath
        value: "/var/openebs/local"
      - name: EXT4Quota
        enabled: "true"
        data:
          softLimitGrace: "0%"
          hardLimitGrace: "0%"
```

If we exclude annotations during upgrade quota enforcement is not active anymore. This could also happen in other use cases.